css-view-transitions-2/Overview.bs

```diff
@@ -517,17 +517,25 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 					We explicitly clear it here since the old Document may be cached by the UA.
 
 			1. [=Queue a global task=] on the [=DOM manipulation task source=] given |newDocument|'s [=relevant global object=],
-				to perform the following step:
+				to perform the following steps:
 
-				1. Let |newDocument|'s [=active view transition=] be a new {{ViewTransition}} in |newDocument|'s [=relevant Realm=],
+				1. Let |inboundTransition| be a new {{ViewTransition}} in |newDocument|'s [=relevant Realm=],
 					whose [=ViewTransition/named elements=] is |outboundTransition|'s [=ViewTransition/named elements=],
 					[=ViewTransition/initial snapshot containing block size=] is |outboundTransition|'s [=ViewTransition/initial snapshot containing block size=],
 					and whose [=ViewTransition/is inbound cross-document transition=] is true.
 
-				1. [=Call the update callback=] for |newDocument|'s [=active view transition=].
+				1. Let |newDocument|'s [=active view transition=] be |inboundTransition|.
+
+				1. [=Call the update callback=] for |inboundTransition|.
 
 				1. Call |onReady|.
 
+				1. At any given time, the UA may decide to skip |inboundTransition|, e.g. after an implementation-defined timeout, by running the following steps:
```

Comment:
We likely need a better term than "At any given time". For example, it can't do it during layout for example, so it's only at a time that a script task can run? I'm not sure if idle time is the right thing or if there is a better term

Reply:
Script tasks can run during layout. But we can make it so that the steps queue a task.